packages/main/src/features/InputSuggestions.js

```diff
@@ -119,6 +119,7 @@ class Suggestions {
 		this.responsivePopover = await this._respPopover();
 		this._beforeOpen();
 		this.responsivePopover.open(this._getComponent());
+		this.responsivePopover.style.maxWidth = this._getComponent().style.width;
```

Comment:
Better to be done in the InputPopover.hbs file (in the Input.js there is already a provate proeprty _inputWidth that can be used)

```js
get styles() {
		return {
			suggestionPopover: {
				"max-width": `${this._inputWidth}px`,
			},
```